spec/Procedures.tex

```diff
@@ -1269,6 +1269,12 @@ \subsection{Determining More Specific Functions}
 \item
  If $F1$ is generic over all types and $F2$ is not generic over all
  types, $M_2$ is more specific.
+ \item
+ If $F1$ is partially concrete generic and $F2$ is generic and not
```

Comment:
I think "partially concrete" and "generic" need to be more separated.  Something about the three-word combination is feeling like not proper English, I suspect because it's two descriptors with an unclear line between them rather than three descriptors or one big one.

I would suggest making it "If $F1$ is partially concrete and generic, and $F2$ is generic and not partially concrete, then $M_1$ is more specific."  And similarly change the next sentence.